tests/unit/backend/chalice/api_server/test_v1_auth.py

```diff
@@ -102,12 +102,19 @@ def restore_path(p):
         self.addCleanup(restore_path, old_path)
         from corpora.common.corpora_config import CorporaAuthConfig
 
-        # Use the CorporaAuthConfig used by the chalice app
-
+        # Configure the CorporaAuthConfig used by the chalice app
         self.auth_config = CorporaAuthConfig()
-        self.auth_config._config["api_base_url"] = f"http://localhost:{PORT}"
-        self.auth_config._config["callback_base_url"] = "http://localhost:5000"
-        self.auth_config.update_defaults()
+        self.auth_config.set(
```

Comment:
NOTE: This is still using the local python mocked oauth server, not the oidc container